pkgs/servers/gerbera/default.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv, fetchgit
+, cmake, pkg-config
+# required
+, libupnp, libuuid, pugixml, libiconv, sqlite, zlib, spdlog, fmt
+, pkgs
+# options
+, enableDuktape ? true
+, enableCurl ? true
+, enableTaglib ? true
+, enableLibmagic ? true
+, enableLibmatroska ? true
+, enableAvcodec ? false
+, enableLibexif ? true
+, enableExiv2 ? false
+, enableFFmpegThumbnailer ? false
+, enableInotifyTools ? true
+}:
+
+with stdenv.lib;
+let
+  optionOnOff = option: if option then "on" else "off";
+in stdenv.mkDerivation rec {
+  pname = "gerbera";
+  version = "1.5.0";
+
+  src = fetchgit {
+    url = meta.repositories.git;
+    rev = "0c486fc53b097227db2443c738647690635bdf99";
+    sha256 = "sha256:19370aagyqyik5dx3ymr0l4sqgwsa5bv8n25lcdcdg1vm2avq0y1";
+  };
+
+  cmakeFlags = [
+    "-DWITH_JS=${optionOnOff enableDuktape}"
+    "-DWITH_CURL=${optionOnOff enableCurl}"
+    "-DWITH_TAGLIB=${optionOnOff enableTaglib}"
+    "-DWITH_MAGIC=${optionOnOff enableLibmagic}"
+    "-DWITH_MATROSKA=${optionOnOff enableLibmatroska}"
+    "-DWITH_AVCODEC=${optionOnOff enableAvcodec}"
+    "-DWITH_EXIF=${optionOnOff enableLibexif}"
+    "-DWITH_EXIV2=${optionOnOff enableExiv2}"
+    "-DWITH_FFMPEGTHUMBNAILER=${optionOnOff enableFFmpegThumbnailer}"
+    "-DWITH_INOTIFY=${optionOnOff enableInotifyTools}"
+    # systemd service will be generated alongside the service
+    "-DWITH_SYSTEMD=OFF"
+  ];
+
+  nativeBuildInputs = [ cmake pkg-config ];
+
+  buildInputs = [
+    libupnp libuuid pugixml libiconv sqlite zlib fmt.dev
+    spdlog
+  ]
+  ++ optionals enableDuktape [ pkgs.duktape ]
+  ++ optionals enableCurl [ pkgs.curl ]
+  ++ optionals enableTaglib [ pkgs.taglib ]
+  ++ optionals enableLibmagic [ pkgs.file ]
+  ++ optionals enableLibmatroska [ pkgs.libmatroska pkgs.libebml ]
+  ++ optionals enableAvcodec [ pkgs.libav.dev ]
+  ++ optionals enableLibexif [ pkgs.libexif ]
+  ++ optionals enableExiv2 [ pkgs.exiv2 ]
+  ++ optionals enableInotifyTools [ pkgs.inotify-tools ]
+  ++ optionals enableFFmpegThumbnailer [ pkgs.ffmpegthumbnailer ];
+
+
+  meta = with stdenv.lib; {
+    homepage = https://docs.gerbera.io/;
+    repositories.git = https://github.com/gerbera/gerbera;
```

Comment:
Is `meta.repositories` a standard? I've never seen it anywhere else.

Reply:
I don't know about a standard, but I have seen it within the nixpkgs repository [1]

(must come from the mediatomb which is within the list [1])

[1] https://github.com/NixOS/nixpkgs/search?q=meta.repositories